script.js

```diff
@@ -1,12 +1,133 @@
 //You can edit ALL of the code here
+const allEpisodes = getAllEpisodes().map(episode =>{
+  const formattedSeason = (episode.season.toString()).padStart(2, "0") 
+  const formattedNumber = (episode.number.toString()).padStart(2, "0")
+  const episodeCorrectFormatV2 = `S${formattedSeason}E${formattedNumber}`
+
+  return {
+    ...episode,
+    formattedNumber: episodeCorrectFormatV2
+  }
+});
+const rootEl = document.getElementById("container")
+const selectEl = document.getElementById("selectEl")
+const headerEl = document.getElementById("headerEl")
+const searchBar = document.getElementById("searchBar")
+const selector = document.getElementById("selector")
+const counter = document.getElementById("episodeCounter")
+
+
```

Comment:
I like the way you declare all variables in one place.

Reply:
Thanks Zaw